src/theory/strings/regexp_entail.cpp

```diff
@@ -799,9 +799,20 @@ bool RegExpEntail::regExpIncludes(Node r1,
   }
   // first, check some basic inclusions
   bool ret = false;
-  bool retSet = false;
-  Kind k1 = r1.getKind();
   Kind k2 = r2.getKind();
+  // if the right hand side is a constant string, this is a membership test
```

Comment:
shouldn't this be symmetrical, i.e., test also for the case where r1 is constant?

Reply:
The method isnt symmetrical since it is checking r2 inclusion in r1.  If r1 is a constant (singleton) regular language, then it only includes r2 if they are equal, which is noted on line 858.

Reply:
Oh, I see. Thanks!